### DISCUSSION TITLE
Migrating to Decky version from non-Decky version

### DISCUSSION THREAD

**Discussion by JpTiger** [moraroy/NonSteamLaunchersDecky#24] [Q&A] [answered]
Hi there! Longtime NSL user here, thank you for your work, and congrats on getting it into the Decky Loader! I'm thinking of moving over to the Decky version for automatic updates etc. Any reason I shouldn't (or does it really not provide any other benefits)? Should I uninstall anything from the vanilla version first? Thanks again

**Comment by moraroy** [comment, marked as answer]
Just simply install the plugin and your good to go, that's about it! I recommend it over the desktop version, the plugin itself is able to be used in Desktop mode as seen here. Currently if you want to take advantage of the Bacup Saves for your games, you would need to run the script aka re-install a launcher. This won't effect or break anything. here.https://youtu.be/CJKyaPIeeKs?si=WY55JQk-1NfG-BGO @JpTiger
